include/aspect/particle/particle.h

```diff
@@ -166,16 +172,13 @@ namespace aspect
 #endif
 
         /**
-          * Resize the properties member variable to hold the number of doubles
-          * required to represent this particle's properties. This function
-          * is called by the PropertyManager class to prepare this particle
-          * for the initialization of its properties.
-          *
-          * @param [in] n_components Number of additional property components
-          * that this particle holds.
-          */
-        void
-        set_n_property_components (const unsigned int n_components);
+         * Destructor. Releases the property handle if it is valid, and
+         * therefore frees that memory space for other particles. (Note:
+         * the memory is managed by the property_pool, and the memory is not
+         * deallocated by this function, it is kept in reserve for other
+         * particles).
```

Comment:
I would remove this parenthetical comment. It comments on an implementation detail of the property pool, but the property pool may implement an entirely different strategy -- which, however, is not something the particle class will actually care about.